src/Spice86/Views/RegistersView.axaml

```diff
@@ -0,0 +1,163 @@
+                        <DataTemplate>
+                            <Grid ColumnDefinitions="Auto,*" Margin="0,2">
+                                <TextBlock Grid.Column="0"
+                                           Text="{Binding Name}"
+                                           FontFamily="Consolas,Menlo,Monaco,monospace"
+                                           Width="40"
+                                           Foreground="#78C9B0" />
+                                <StackPanel Grid.Column="1" Orientation="Horizontal">
+                                    <!-- General purpose 32-bit registers (EAX, EBX, etc.) upper word -->
+                                    <SelectableTextBlock Text="{Binding UpperWordHex}"
+                                                         FontFamily="Consolas,Menlo,Monaco,monospace"
+                                                         behaviors:HighlightBehavior.IsHighlighted="{Binding UpperWordChanged}" />
+
+                                    <!-- High byte (AH, BH, etc.) -->
+                                    <SelectableTextBlock Text="{Binding HighByteHex}"
+                                                         FontFamily="Consolas,Menlo,Monaco,monospace"
+                                                         behaviors:HighlightBehavior.IsHighlighted="{Binding HighByteChanged}"
+                                                         Margin="4,0,0,0" />
+
+                                    <!-- Low byte (AL, BL, etc.) -->
+                                    <SelectableTextBlock Text="{Binding LowByteHex}"
+                                                         FontFamily="Consolas,Menlo,Monaco,monospace"
+                                                         behaviors:HighlightBehavior.IsHighlighted="{Binding LowByteChanged}"
+                                                         Margin="4,0,0,0" />
+                                </StackPanel>
+                            </Grid>
+                        </DataTemplate>
+                    </ItemsControl.ItemTemplate>
+                </ItemsControl>
+            </StackPanel>
+        </Border>
+
+        <!-- Pointer Registers -->
+        <Border Grid.Row="2"
+                BorderBrush="{DynamicResource SemiColorBorder}"
+                BorderThickness="1"
+                CornerRadius="4"
+                Padding="8"
+                Margin="0,0,0,8">
+            <StackPanel>
+                <ItemsControl ItemsSource="{Binding PointerRegisters}">
+                    <ItemsControl.ItemTemplate>
+                        <DataTemplate>
+                            <Grid ColumnDefinitions="Auto,*" Margin="0,2">
+                                <TextBlock Grid.Column="0"
+                                           Text="{Binding Name}"
+                                           FontFamily="Consolas,Menlo,Monaco,monospace"
+                                           Width="40"
```

Comment:
MinWidth should be used instead of fixed Height/Width to allow for responsive design.

Reply:
Done.